.github/workflows/all.yml

```diff
@@ -123,11 +124,26 @@ jobs:
               https://github.com/build-trust/.github/blob/main/CONTRIBUTING.md#commit-messages
             ' && exit 1)
 
+      - name: Check If PR Author Made Changes Only To CONTRIBUTORS.md
+        run: |
+          paths_updated=$(git diff --name-only origin/develop..HEAD)
+
+          if echo "$paths_updated" | grep $CONTRIBUTORS_CSV_PATH; then
+            # user has made changes to the CONTRIBUTORS.md file, we need to ensure that PR
+            # is only accepting the CLA
+            no_paths_updated=$(echo $paths_updated | wc -l)
+            if [[ $no_paths_updated -gt 1 ]]; then
+              echo "
+                CLA is required for all contributors.
+                Please create a pull request only accepting the CLA by adding your Github details in a row at the end of the CONTRIBUTORS.csv file.
+                Contributors.csv file is located at: $CONTRIBUTORS_CSV_PATH
```

Comment:
```suggestion
                Please create a new pull request to accept the CLA by adding your Github details in a row at the end of the CONTRIBUTORS.csv file.
                This new pull request must only change the CONTRIBUTORS.csv file.
                CONTRIBUTORS.csv file is located at: $CONTRIBUTORS_CSV_PATH
```

Reply:
@mrinalwadhwa if further questions need to be asked by the PR authors concerning CLA, where would be the best place?
- The current failing PR?
- A Github discussion thread that's created to help new contributors
- Our Discord channel

Reply:
@mrinalwadhwa should we add the github discussion thread in the error message? Is there a github discussion thread

Reply:
@mrinalwadhwa is https://github.com/build-trust/ockam/discussions/1642 fine?

Reply:
@metaclips @mrinalwadhwa Here is a new github discussion link that we can use.
https://github.com/build-trust/ockam/discussions/6112

TODOs for this link:
- Add the video for CLA acceptance using github.com UI
- Add link to contributor guide https://github.com/build-trust/.github/blob/main/CONTRIBUTING.md

Reply:
I'll push a follow up PR that updates the contributors.csv link when this PR is merged